web/sites/default/config/search_api.index.documents.yml

```diff
@@ -147,6 +147,7 @@ processor_settings:
   entity_status: {  }
   hierarchy:
     fields:
+      field_display_groups: group-field_bureaus
```

Comment:
I'm not certain this should have changed. You aren't touching documents in this story and the documents search index doesn't have any specific group relationships that should have been impacted by any of the other changes.

Reply:
I think wires got crossed when moving previous work. I've removed `field_display_groups`.

Reply:
Similar to the previous comments. You should only do this within the UI, then save, then run a `drush cex` to get the change.

Reply:
Please don't mark comments as resolved if the task is to remove the file. It will automatically resolve the comment when the file is removed. It's too easy to miss these important tasks if the comment is resolved.